### PR TITLE
Added Github Actions - PR Builder

### LINKED ISSUE #22: Create Pull Request Builder - Github Actions
Create pull request builder to validate PRs

### PR DESCRIPTION
**Pull request by AbhiKul10**
Created Pull Request Builder to validate PRs for the Project

Screenshot 2021-06-27 at 4 27 47 PM

- Added Github Actions
- Dependencies install and Lint Check

Fixes #22

### CODE REVIEW

File: src/components/HeaderBar/HeaderBar.js
Comment (shehand) [inline src/components/HeaderBar/HeaderBar.js:1]: Use double quotes instead of single

File: src/components/HeaderBar/styles.js
Comment (shehand) [inline src/components/HeaderBar/styles.js:1]: revert to double quotes

File: src/components/LoginForm/LoginForm.js
Comment (shehand) [inline src/components/LoginForm/LoginForm.js:25]: double quotes for string also

File: src/components/LoginForm/styles.js
Comment (shehand) [inline src/components/LoginForm/styles.js:73]: double quotes

File: src/components/SignupForm/SignupForm.js
Comment (shehand) [inline src/components/SignupForm/SignupForm.js:46]: double quotes

File: src/config/config.example.js
Comment (shehand) [inline src/config/config.example.js:15]: Just for curiosity, what this variable does?

File: src/screens/AnswerScreen/AnswerScreen.js
Comment (shehand) [inline src/screens/AnswerScreen/AnswerScreen.js:4]: I think we can remove this from here. Is it?

File: src/screens/AnswerScreen/styles.js
Comment (shehand) [inline src/screens/AnswerScreen/styles.js:27]: double quotes

File: src/screens/HomeScreen/HomeScreen.js
Comment (shehand) [inline src/screens/HomeScreen/HomeScreen.js:3]: Same question

File: src/screens/OnboardingScreen/OnboardingScreen.js
Comment (shehand) [inline src/screens/OnboardingScreen/OnboardingScreen.js:9]: Is is possible to change the color codes in to one method? either # or rgb and not both in the same line.

File: src/screens/ProfileScreen/ProfileScreen.js
Comment (shehand) [inline src/screens/ProfileScreen/ProfileScreen.js:4]: same question

File: src/screens/ProfileScreen/ProfileScreen.js
Comment (shehand) [inline src/screens/ProfileScreen/ProfileScreen.js:66]: If this 'profile_picture.png' string is having multiple occurrences on this same file, replace it with a constant. otherwise please ignore this comment.

File: .github/workflows/build.yml
Comment (shehand) [inline .github/workflows/build.yml:16]: So currently we only consider the linting issues. My suggestion is to hold this until we have some test that can be executed during the build. This action will not actually reflect the requirement of a builder. @Wathsara need some clarification on this.

**Review by shehand — CHANGES_REQUESTED** [review]
I'm confused with this PR @AbhiKul10. Most of the changes I saw through this PR are fixing linting issues/suggestions and they are actually not related to the original title of this PR. So I think it is better to open a separate issue to address the linting issues and push them by using another PR. However, I added some review comments. Address them first.

File: .github/workflows/build.yml
Comment (Wathsara) [reply .github/workflows/build.yml:16]: @shehand Since we dont have any test cases and @AbhiKul10 still works on that, hope this initial step is fine. Once we have tests We can update the action.

File: .github/workflows/build.yml
Comment (Wathsara) [inline .github/workflows/build.yml:5]: @AbhiKul10 this should be for a Pull request.

**Review by Wathsara — CHANGES_REQUESTED** [review]

File: src/components/HeaderBar/HeaderBar.js
Comment (AbhiKul10) [reply src/components/HeaderBar/HeaderBar.js:1]: Sure! I'll change the linting rules and set Double quotes instead of single quotes

File: src/config/config.example.js
Comment (AbhiKul10) [reply src/config/config.example.js:15]: typo

File: src/screens/AnswerScreen/AnswerScreen.js
Comment (AbhiKul10) [reply src/screens/AnswerScreen/AnswerScreen.js:4]: It was added by the linter itself. These lines are disabling some rules of linter that are causing a change in code!

File: src/screens/ProfileScreen/ProfileScreen.js
Comment (AbhiKul10) [reply src/screens/ProfileScreen/ProfileScreen.js:66]: profile_picture.png is only used once for the path only

File: src/screens/OnboardingScreen/OnboardingScreen.js
Comment (AbhiKul10) [reply src/screens/OnboardingScreen/OnboardingScreen.js]: Done

File: .github/workflows/build.yml
Comment (AbhiKul10) [reply .github/workflows/build.yml]: @Wathsara changed it for pull request, but I have a doubt like when changing it for pull_request, are we required to mention the branch also?

File: .github/workflows/build.yml
Comment (AbhiKul10) [reply .github/workflows/build.yml:16]: Yes @shehand ! I am on it and I'll update the actions too for the same.

File: .github/workflows/build.yml
Comment (Wathsara) [reply .github/workflows/build.yml]: @AbhiKul10 Yes we have to

**Comment by AbhiKul10** [comment]
@shehand @Wathsara Please review

**Review by shehand — CHANGES_REQUESTED** [review]
As I said earlier, this PR should be separated into two distinct PRs, one for the GH action and one for linter suggestions.

**Comment by AbhiKul10** [comment]
Okay @shehand I am removing all the linting from this Pull request and will open a new issue and PR regarding linting. Please merge another PR first otherwise workflow might fail.  I hope I am doing it the right way!

**Review by shehand — APPROVED** [review]

**Comment by shehand** [comment]
@AbhiKul10, there is a conflict issue in the PR. Could you please take a look into this?

**Comment by AbhiKul10** [comment]
@shehand @Wathsara Can you please give me write access in order to resolve merge conflicts

**Comment by shehand** [comment]
> @shehand @Wathsara Can you please give me write access in order to resolve merge conflicts, as **Resole conflicts** button is disabled.

Done. Are you managing the Kanban board accordingly? I can't see the labels and status on Issues and PRs.

**Comment by AbhiKul10** [comment]
> > @shehand @Wathsara Can you please give me write access in order to resolve merge conflicts, as **Resole conflicts** button is disabled.
> 
> Done. Are you managing the Kanban board accordingly? I can't see the labels and status on Issues and PRs.

Earlier I didn't have the appropriate role to access the kanban board, was only able to label the issues. Now I am able to create cards and manage them when you gave me write access. Now onwards I'll manage it.

**Comment by AbhiKul10** [comment]
> > > @shehand @Wathsara Can you please give me write access in order to resolve merge conflicts, as **Resole conflicts** button is disabled.
> > 
> > 
> > Done. Are you managing the Kanban board accordingly? I can't see the labels and status on Issues and PRs.
> 
> Earlier I didn't have the appropriate role to access the kanban board, was only able to label the issues. Now I am able to create cards and manage them when you gave me write access. Now onwards I'll manage it.

**Comment by AbhiKul10** [comment]
@shehand still, my Resolve conflict button is disabled

**Comment by shehand** [comment]
> @shehand still, my Resolve conflict button is disabled

It's disable for me also. You need to resolve this locally and push via a PR.

**Review by shehand — APPROVED** [review]

**Pull request merged**
